packages/devnet/README.md

```diff
@@ -0,0 +1,75 @@
+## 0x Devnet
```

Comment:
Hm, are we sold on this name?

Reply:
I like it because "testnet" has a generally accepted meaning that is slightly different. A testnet is typically open to the public, has some faucet mechanisms, etc. I'm open to changing it though.